content/applications/finance/accounting/others/analytic/analytic_accounting.rst

```diff
@@ -0,0 +1,115 @@
+- :guilabel:`Currency`: select the currency of the :guilabel:`Analytic Account`.
+
+Then, fill in your budget information.
+
+.. seealso::
+   :doc:`Financial budget <../adviser/budget>`.
+
+Analytic plans
+==============
+
+You can access the :guilabel:`Analytic Plans` by going to :menuselection:`Accounting -->
+Configuration --> Analytic Accounting: Analytic Plans`. Click :guilabel:`New` to create a new plan.
+
+.. image:: analytic_accounting/analytic_plans.png
+   :align: center
+   :alt: create an analytic plan
+
+The following information must be completed:
+
+- :guilabel:`Parent`: link you plan to another :guilabel:`Analytic Plan` to build a hierarchy
+  between your plans;
+- :guilabel:`Default Applicability`: :guilabel:`Optional`, :guilabel:`Mandatory`, or
+  :guilabel:`Unavailable`;
+- :guilabel:`Color`: select the color of the tag related to this specific plan;
+- :guilabel:`Company`: add the company to which the plan applies;
+- :guilabel:`Domain`: choose where your plan is applicable;
+- :guilabel:`Financial Accounts Prefix`: select the prefix of the account(s) with which this plan
+  should be applied;
+- :guilabel:`Product Category`: decide to which product category the plan applies;
+
+- :guilabel:`Applicability`:
+   - :guilabel:`Optional`: if selected, it is not mandatory to add the analytic plan in the widget;
+   - :guilabel:`Mandatory`: if selected, an orange bullet is visible in the widget next to the plan
+     until the analytic distribution is done (the bullet then turns to green); it is not possible
+     to confirm the entry if no analytic account is selected;
+   - :guilabel:`Unavailable`: if selected, the plan is not available in the widget.
+
+Two smart buttons are available in the right top corner:
+
+   - :guilabel:`Subplans`: can be created to have a more complex analytic structure. Click the
+     :guilabel:`Subplans` smart button, and then :guilabel:`New` to add a subplan;
+   - :guilabel:`Analytic Accounts`: to reach the subplans related to the plan.
+
+.. note::
+   - The analytic widget is prefilled based on the :guilabel:`Applicability`, and the
+     :guilabel:`Analytic Distribution Models`;
+   - Each :guilabel:`Analytic plan` must have at least one :guilabel:`Analytic Account`.
+
+Analytic distribution
+=====================
+
+When creating an invoice or a bill it is mandatory to add a plan in the
+:guilabel:`Analytic Distribution` column. A **widget** opens. Fill in the different information. You
+can add **tags** to reflect the related :guilabel:`Analytic Accounts`, and decide how to split the
+costs between the accounts by modifying the percentage.
+
+.. image:: analytic_accounting/analytic_distribution.png
+   :align: center
+   :alt: create a distribution template
+
+Analytic distribution models
+----------------------------
+
+The :guilabel:`Analytic Distribution Models` are templates that can be created by going to
```

Comment:
```suggestion
The :guilabel:`Analytic Distribution Models` are templates that can be created by going to
```
Mention somewhere that it is automated if you're using templates, I'll let check the video again